src/main/kotlin/com/terraformation/backend/nursery/daily/NurseryNotifier.kt

```diff
@@ -0,0 +1,35 @@
+package com.terraformation.backend.nursery.daily
+
+import com.terraformation.backend.customer.model.FacilityModel
+import com.terraformation.backend.daily.FacilityNotifier
+import com.terraformation.backend.db.default_schema.FacilityId
+import com.terraformation.backend.db.default_schema.FacilityType
+import com.terraformation.backend.log.perClassLogger
+import com.terraformation.backend.nursery.db.BatchStore
+import java.time.LocalDate
+import javax.inject.Named
+import org.springframework.context.ApplicationEventPublisher
+
+@Named
+class NurseryNotifier(
+    private val batchStore: BatchStore,
+    private val eventPublisher: ApplicationEventPublisher,
+) : FacilityNotifier {
+  private val log = perClassLogger()
+
+  override fun sendNotifications(facility: FacilityModel, todayAtFacility: LocalDate) {
+    if (facility.type == FacilityType.Nursery) {
+      log.info("Generating seedling batch notifications for facility ${facility.id}")
+
+      if (facility.lastNotificationDate != null) {
+        seedlingBatchReady(facility.id, facility.lastNotificationDate, todayAtFacility)
+      }
+    }
+  }
+
+  private fun seedlingBatchReady(facilityId: FacilityId, after: LocalDate, until: LocalDate) {
```

Comment:
seedlings plural?

Reply:
I think we use the term "seedling batch" (singular "seedling") in the UI, no? Or did we change to plural?

Reply:
We use `Seedlings Batches` as a title, and `Seedling Batch` as the column header (referring to single batches). I guess it is correct here, my mistake.